composer.json

```
@@ -13,14 +13,15 @@
     "minimum-stability": "stable",
     "require": {
         "php": ">=7.0.0",
-        "guzzlehttp/guzzle": "^6.3",
-        "illuminate/support": "~5.0|~6"
+        "guzzlehttp/guzzle": "^6.3|^7.0",
+        "illuminate/support": "~5.0|~6.0|~7.0|~8.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.0",
+        "phpunit/phpunit": "^6.0|^7.0|^8.0",
         "squizlabs/php_codesniffer": "^3.0",
-        "orchestra/testbench": "~3.0",
-        "mockery/mockery": "^0.9.7"
+        "orchestra/testbench": "^6.0",
```

This should reflect each version of Laravel above.

```suggestion
        "orchestra/testbench": "^3.0|^4.0|^5.0|^6.0",
```